src/api/app/assets/javascripts/webui2/package-view_file.js

```diff
@@ -0,0 +1,30 @@
+var DAYS = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+var MONTHS = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+
+// The jshint ignore:line is needed to ignore the offense: `'addChangesEntryTemplate' is defined but never used.` This is used in the view directly
```

Comment:
Is this not a bug? I mean... it is normal you define Javascript methods to be used in the view... :confused:

Reply:
I don't know... the code could be changed to avoid this, but I don't have a strong opinion on this.

Reply:
but I meant that without changing the code, this shouldn't fail. Or am I missing something? :confused:

Reply:
I don't know much about `jshint`, but I think it shouldn't make this as an offense.

Reply:
have you tried if it fails if you remove the comment? If it does, I would say it should be reported to jshint

Reply:
Removing this comment shouldn't have any effect,  is only a comment for line 6

Reply:
The comment is only to explain why I put the ignore on the next line.